.circleci/config.yml

```diff
@@ -35,7 +26,13 @@ e2eTest: &e2eTest
     - run:
         name: Run test
         command: |
-          E2E_KUBECONFIG=/e2e/kubeconfig ./e2e-harness test --test-dir=${TEST_DIR}
+          docker run --rm --network host \
+            -v /home/circleci/.go_workspace/src/github.com/giantswarm/kubernetes-nginx-ingress-controller/${TEST_DIR}:/e2e \
```

Comment:
Do you need to specify this whole path `-v /home/circleci/.go_workspace/src/github.com/giantswarm` ? Don't you have `kubernetes-nginx-ingress-controller` in your working directory already?

Reply:
Yeah, it seem absolute path has to be used, relative didn't work.

Reply:
Then a nitpick maybe, adding `$(pwd)` instead of this long path

Reply:
Fixed